account_payment_notification/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+{
+    "name": "Account payment notification",
+    "summary": "Notifiy upcoming payments",
+    "version": "15.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Accounting",
+    "website": "https://github.com/OCA/account-payment",
+    "author": "Moduon, Odoo Community Association (OCA)",
+    "maintainers": ["yajo"],
+    "license": "LGPL-3",
+    "installable": True,
+    "depends": [
+        "account",
+        "sms",
```

Comment:
This dependency and its code should be extracted in a glue module for not making it dependent of such ugly modules and all the IAP infrastructure.

Reply:
The fact SMS is installed doesn't mean you have to use it or pay for it. Actually the sms module is open source and allows partners to configure a different provider than Odoo IAP. This module is shipped with SMS notifications disabled by default. But it's part of its core logic and we have no plans or needs to split it.

Reply:
I know, but it's a maintenance burden and doesn't follow usual Odoo modularity, but you can do what you consider.

Reply:
Thanks for the suggestion, I understand it. However, for now we'll keep it as it is. The split can be done later by another member that finds that as a requirement, or by ourselves if it becomes a real burden.